db-scripts/src/main/resources/cgds.sql

```diff
@@ -775,6 +791,22 @@ CREATE TABLE `clinical_event_data` (
 );
 
 -- --------------------------------------------------------
+DROP TABLE IF EXISTS `reference_genome_gene`;
+CREATE TABLE `reference_genome_gene` (
+    `ENTREZ_GENE_ID` int(11) NOT NULL,
+    `CYTOBAND` varchar(64) NOT NULL,
+    `EXONIC_LENGTH` int(11) DEFAULT NULL,
+    `GENE_START` bigint(20) DEFAULT NULL,
+    `GENE_END` bigint(20) DEFAULT NULL,
+    `GENE_STABLE_ID` varchar(64) DEFAULT NULL,
+    `CHR` varchar(64) DEFAULT NULL,
```

Comment:
Should we normalize these lengths with copy_number_seg, CHR?

Reply:
changed to chr(5), thanks!